cherrypy/lib/cpstats.py

```diff
@@ -308,6 +308,13 @@ def _get_threading_ident():
     return threading._get_ident()
 
 
+def _get_response_status(resp):
+    response_status = getattr(resp, 'output_status', None) or resp.status
```

Comment:
it's probably `response_status = getattr(resp, 'output_status', resp.status)`, though I'd prefer try/except approach